code/modules/karma/karma_holder.dm

```diff
@@ -390,8 +391,6 @@
 					switch(href_list["KarmaBuy"])
 						if("1")
 							karma_purchase(5, "job", "Barber")
-						if("2")
```

Comment:
this wont cause any problems with the option values being 1, 3, 4 right?

Reply:
Nope